nisystemlink/clients/webapp/_webapp_client.py

```diff
@@ -0,0 +1,51 @@
+"""Implementation of WebappClient."""
+
+from typing import Any, Optional
+
+from nisystemlink.clients import core
+from nisystemlink.clients.core._uplink._base_client import BaseClient
+from nisystemlink.clients.core._uplink._methods import get, post
+from uplink import Body
+
+from . import models
+
+
+class WebappClient(BaseClient):
+    def __init__(self, configuration: Optional[core.HttpConfiguration] = None):
+        """Initialize an instance.
+
+        Args:
+            configuration: Defines the web server to connect to and information about
+                how to connect. If not provided, an instance of
+                :class:`JupyterHttpConfiguration <nisystemlink.clients.core.JupyterHttpConfiguration>`
+                is used.
+
+        Raises:
+            ApiException: if unable to communicate with the WebApp Service.
+        """
+        if configuration is None:
+            configuration = core.JupyterHttpConfiguration()
+
+        super().__init__(configuration, base_path="/niapp/v1/webapps/")
+
+    @post("query", args=[Body])
+    def query_webapps(
+        self, query: models.WebAppsAdvancedQuery
+    ) -> models.WebAppsResponse:
+        """Use the Dynamic Linq query language to specify filters for webapps.
+        An empty request body queries all webapps.
+        """
+        ...
+
+    @get("{id}/content")
+    def get_content(self, id: str) -> Any:
```

Comment:
It's been awhile since I looked at the uplink docs, but with the `Any` type this will return the whole `requests.Response` object, right? We don't do that anywhere else. Maybe we can reuse the `IteratorFileLike` object that we created for the export method of the DFS client?https://github.com/ni/nisystemlink-clients-python/blob/3459a508e43512b4e747d509c3ba58fd4c8f9aa5/nisystemlink/clients/dataframe/_data_frame_client.py#L305